one-ui/src/test/java/com/marklogic/hub/oneui/controllers/SavedQueriesControllerTest.java

```diff
@@ -0,0 +1,215 @@
+package com.marklogic.hub.oneui.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.marklogic.client.FailedRequestException;
+import com.marklogic.client.document.GenericDocumentManager;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.hub.DatabaseKind;
+import com.marklogic.hub.oneui.Application;
+import com.marklogic.hub.oneui.TestHelper;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {Application.class})
+public class SavedQueriesControllerTest extends TestHelper {
```

Comment:
I emailed Ryan and Srikanth about this - I'd rather see all these tests against the data service and thus the tests should be in the core project, not in one-ui. The core project is its own thing, and we want to verify its functionality within the core project. We don't want to depend on a UI project to verify it.

Additionally, I don't think testing the controller proves much of anything, since the controller only needs to construct the appropriate DatabaseClient. 

No action yet on this, waiting to hear back from Ryan and Srikanth.